.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
Do you have an example scenario where you would want to skip the CI pipeline?

Reply:
No - but you can just add to a commit message either `ci skip` or `skip ci` to prevent this workflow from running, this was guidance from the `auto` docs to prevent unintentional cyclic behaviour on github pipelines.

Reply:
FWIW, the default behaviour we are using here for conventional commits, is to skip unless you use `fix, `feat` or `breaking`. This means that simply using something like `build:` at the start of your commit message should skip building.
https://intuit.github.io/auto/docs/generated/conventional-commits